junifer/configs/juseless/datagrabbers/ixi_vbm.py

```diff
@@ -0,0 +1,91 @@
+"""Provide class for IXI VBM juseless datalad datagrabber."""
+
+# Authors: Federico Raimondo <f.raimondo@fz-juelich.de>
+#          Leonard Sasse <l.sasse@fz-juelich.de>
+#          Synchon Mandal <s.mandal@fz-juelich.de>
+# License: AGPL
+
+from pathlib import Path
+from typing import List, Union
+
+from ....api.decorators import register_datagrabber
+from ....datagrabber import PatternDataladDataGrabber
+from ....utils import raise_error
+
+
+@register_datagrabber
+class JuselessDataladIXIVBM(PatternDataladDataGrabber):
+    """Juseless IXI VBM DataGrabber class.
+
+    Implements a DataGrabber to access the IXI VBM data in Juseless.
+
+    Parameters
+    -----------
+    datadir : str or pathlib.Path, optional
+        The directory where the datalad dataset will be cloned. If None,
+        the datalad dataset will be cloned into a temporary directory
+        (default None).
+    sites : {"Guys", "HH", "IOP"} or list of the
+        options, optional.
+        Which sites to access data from. If None, all available sites are
+        selected (default None).
+    """
+
+    def __init__(
+        self,
+        datadir: Union[str, Path, None] = None,
+        sites: Union[str, List[str], None] = None,
+    ) -> None:
+        """Initialize the class."""
+        uri = "git@jugit.fz-juelich.de:inm7/datasets/datasets_repo.git"
```

Comment:
We need to avoid using the superdataset in Junifer, as this creates problems with subdatasets with the pattern grabber.

By checking the superdataset, we can see that the CAT12.5 data is in another superdataset: http://cat-12.5.ds.inm7.de/alias/super

And indeed the IXI is here: http://cat_12.5.ds.inm7.de/b71/07c52-8408-11ea-89c6-a0369f287950

Can we check with @FeHoff if there is an alias for the IXI directly?

Reply:
I have changed it to the alias of the IXI dataset directly and removed the get_elements() method. Now, the datagrabber seems to derive the correct paths, and elements on its own with the pattern provided, but the tests on juseless fail anyhow. I am also unable to get files from this dataset via the datalad CLI manually, so I guess I dont have access to these files?